internal/authz/authz.go

```diff
@@ -76,12 +75,9 @@ const (
 )
 
 var (
-	ErrUnauthorizedResponse                  = status.Error(codes.Code(openfgav1.AuthErrorCode_forbidden), "the principal is not authorized to perform the action")
-	ErrBadRequestMaxModulesInRequestExceeded = status.Error(codes.Code(openfgav1.AuthErrorCode_forbidden), fmt.Sprintf("the principal cannot write tuples of more than %v module(s) in a single request", MaxModulesInRequest))
```

Comment:
@miparnisari - I feel this one in particular we may need to keep public, otherwise folks will not be able to understand why they're not authorized even though they have access to each module individually

Reply:
Oh. But the way it is today it will be overriden here: https://github.com/openfga/openfga/blob/81e97836a35f0e06530161ebeb9a5894d97c688f/pkg/server/server.go#L907-L911

🤔 feels like we need to change this too somehow.